bin/jgrep

```diff
@@ -18,15 +18,19 @@ def do_grep(json, expression)
     if @options[:field].empty?
         result = JGrep::jgrep((json), expression, nil, @options[:start])
         result = result.slice(@options[:slice]) if @options[:slice]
-        unless result == [] or @options[:quiet] == true
+        if result == [] or @options[:quiet] == true
```

Comment:
A couple of points here:

1. This will always `exit 1` if quiet is enabled, but it should exit 0 if a match is found with quiet enabled.
2. Should this new behaviour be limited to when quiet is used or did you want it without quiet too?  So the default behaviour without quiet is unchanged (always exiting zero) but if you supply -q then exits zero/one?  The help text for quiet suggests it should, but it doesn't.